frontend/src/components/Registration/LoginIndex.tsx

```diff
@@ -3,50 +3,75 @@ import {
   Center,
   Flex,
   FormControl,
+  FormErrorMessage,
   FormLabel,
   Heading,
   Input,
 } from "@chakra-ui/react";
+import { Field, Form, Formik } from "formik";
 import React from "react";
 import { Link } from "react-router-dom";
 import useThemeColor from "../../hooks/useThemeColor";
 
+interface ILoginForm {
+  username: string;
+  password: string;
+}
+
 const LoginIndex = () => {
   const { backgroundColor, borderColor } = useThemeColor();
+
+  const handleOnSubmit = (values: ILoginForm) => {
+    console.log(values);
```

Comment:
don't forget to remove console logs!!

You can benefit from adding a linter to check for this